packages/settings-library/src/settings_library/email.py

```diff
@@ -11,14 +11,15 @@
 class SMTPSettings(BaseCustomSettings):
     """Simple Mail Transfer Protocol"""
 
+    # TODO: NameEmail or EmailStr
```

Comment:
missing todo?

Reply:
I left it as an exercise but did not dare to change even more...